monkey/monkey_island/cc/ui/src/components/ui-components/plugins-marketplace/AvailablePlugins.tsx

```diff
@@ -23,7 +23,6 @@ const AvailablePlugins = () => {
   const {availablePlugins} = useContext(PluginsContext);
```

Comment:
Also, the newly installed plugin object should be reflected in the installedPlugins list